src/main/kotlin/io/snyk/plugin/services/download/CliDownloaderService.kt

```diff
@@ -57,6 +60,15 @@ class SnykCliDownloaderService {
     }
 
     fun downloadLatestRelease(indicator: ProgressIndicator, project: Project) {
+        if (!pluginSettings().automaticCLIUpdatesEnabled) {
+            if (!isCliInstalled()) {
+                val msg =
+                    "The plugin cannot scan without Snyk CLI, but automatic download is disabled. " +
```

Comment:
Do we need to add this info to the whole details panel maybe, since users may miss the ballon notification?

Reply:
@shlomirozilyo

Reply:
@bastiandoetsch what was the answer to this?

Reply:
We said we would add the information (see screenshot) to the setting. This has been done.